src/main/java/com/bootcamp/demo/Controller/AccountController.java

```diff
@@ -45,9 +45,15 @@ public void delete(@PathVariable String username) throws ControllerException {
     public AccountDetails viewAccount(@PathVariable String username) throws ControllerException {
         try {
             Account account = service.findOne(username);
-            AccountDetails accountDetails = new AccountDetails(account.getName(), account.getEmail(), account.getUsername(), account.getPhoneNumber(), account.getAddress(), account.getDateOfBirth());
-            System.out.println("[VIEW]: Account with username " + account.getUsername());
-            return accountDetails;
+            return new AccountDetails(
+                    account.getName(),
+                    account.getEmail(),
+                    account.getUsername(),
+                    account.getPhoneNumber(),
+                    account.getAddress(),
+                    account.getDateOfBirth());
+
+
```

Comment:
I think we should move this to the service layer since it concerns the logic of the program. We could change the service interface to support a new type, something like: interface Service<InputData, OutputData>